binding.gyp

```diff
@@ -24,12 +21,19 @@
             "conditions": [
                 [ "SASL==\"n\"", { "sources!": 
                   ["LDAPSASL.cc", "SASLDefaults.cc"] } ], 
-                [ "SASL==\"y\"", { "sources!": ["LDAPXSASL.cc"] } ]
+                [ "SASL==\"y\"", { "sources!": ["LDAPXSASL.cc"] } ],
+		['OS=="linux" and NODE_VERSION > 9', {
```

Comment:
Does it work for Node 10?

Reply:
It certainly should although i haven't tested it on node10